dev/com.ibm.ws.security.saml.sso_fat.3/fat/src/com/ibm/ws/security/saml/fat/common/PkixSAMLTests.java

```diff
@@ -80,7 +81,7 @@ public void pkixSAMLTests_SpecifiedTrustAnchorTest() throws Exception {
      * The jks file does not have the self-signed certificates from the tfim IdP.
      * So, the test fails
      */
-    @ExpectedFFDC(value = { "com.ibm.ws.security.saml.error.SamlException", "org.opensaml.ws.security.SecurityPolicyException" })
```

Comment:
Removing the ExpectedFFDC concerns me.  It may make sense to have ExpectedFFDC be Repeatable and then update the fattest.simplicity function to handle the repeat and then you can have the right repeat actions go with the right exceptions.

Of course this comment applies to many other classes.  I am not marking every place that this applies, but it should be noted those classes should be changed as well.

Reply:
I will address this fattest.simplicity update and repeatable ExpectedFFDC changes with another issue/pr

Reply:
#17815